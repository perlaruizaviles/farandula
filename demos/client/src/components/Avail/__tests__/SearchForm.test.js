import React from 'react';
import {shallow} from 'enzyme';
import SearchForm from '../SearchForm';

describe('Rendering TravelResults ', () => {

  const tree = shallow(<SearchForm />);
  
  it('Should create an snapshot for TravelResults', () => {
    expect(tree).toMatchSnapshot();
  });
});