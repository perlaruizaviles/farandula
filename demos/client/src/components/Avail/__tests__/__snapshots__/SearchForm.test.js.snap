exports[`Rendering TravelResults  Should create an snapshot for TravelResults 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Connect(Form(SearchForm))
    destroyOnUnmount={true}
    enableReinitialize={false}
    forceUnregisterOnUnmount={false}
    form="SearchForm"
    getFormState={[Function]}
    initialValues={undefined}
    keepDirtyOnReinitialize={false}
    persistentSubmitErrors={false}
    pure={true}
    shouldAsyncValidate={[Function]}
    shouldValidate={[Function]}
    touchOnBlur={true}
    touchOnChange={false}
    validate={[Function]} />,
  "nodes": Array [
    <Connect(Form(SearchForm))
      destroyOnUnmount={true}
      enableReinitialize={false}
      forceUnregisterOnUnmount={false}
      form="SearchForm"
      getFormState={[Function]}
      initialValues={undefined}
      keepDirtyOnReinitialize={false}
      persistentSubmitErrors={false}
      pure={true}
      shouldAsyncValidate={[Function]}
      shouldValidate={[Function]}
      touchOnBlur={true}
      touchOnChange={false}
      validate={[Function]} />,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ReduxForm />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": ReduxForm {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Connect(Form(SearchForm))
          destroyOnUnmount={true}
          enableReinitialize={false}
          forceUnregisterOnUnmount={false}
          form="SearchForm"
          getFormState={[Function]}
          initialValues={undefined}
          keepDirtyOnReinitialize={false}
          persistentSubmitErrors={false}
          pure={true}
          shouldAsyncValidate={[Function]}
          shouldValidate={[Function]}
          touchOnBlur={true}
          touchOnChange={false}
          validate={[Function]} />,
        "_debugID": 2,
        "_renderedOutput": <Connect(Form(SearchForm))
          destroyOnUnmount={true}
          enableReinitialize={false}
          forceUnregisterOnUnmount={false}
          form="SearchForm"
          getFormState={[Function]}
          initialValues={undefined}
          keepDirtyOnReinitialize={false}
          persistentSubmitErrors={false}
          pure={true}
          shouldAsyncValidate={[Function]}
          shouldValidate={[Function]}
          touchOnBlur={true}
          touchOnChange={false}
          validate={[Function]} />,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ReduxForm />,
}
`;
